### PR TITLE
DOMA-3349 Custom iframe title for billing integration

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/miniapp/components/AppIndex/IndexBillingAppPage.tsx [hunk 1]
```diff
@@ -57,23 +59,39 @@ export const IndexBillingAppPage: React.FC<IndexBillingAppPageProps> = ({ id })
     }
 
     const PageTitle = get(integration, 'name', BillingMessage)
+
+    if (PageTitle && title === BillingMessage) {
```
Comment (pahaz) [inline apps/condo/domains/miniapp/components/AppIndex/IndexBillingAppPage.tsx:63]: Look strange. Why it so?

File: apps/condo/domains/miniapp/components/AppIndex/IndexBillingAppPage.tsx [hunk 2]
```diff
@@ -57,23 +59,39 @@ export const IndexBillingAppPage: React.FC<IndexBillingAppPageProps> = ({ id })
...
+        setTitle(PageTitle)
```
Comment (sitozzz) [inline apps/condo/domains/miniapp/components/AppIndex/IndexBillingAppPage.tsx:64]: Change state inside render block of component without any hooks looks like potential unnecessary render

Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/components/AppIndex/IndexBillingAppPage.tsx]: Refactored 😄 👍

**Review by pahaz — APPROVED** [review]

**Comment by pahaz** [comment]
I understand. You want to add the title customization by mini-app

**Pull request merged**